config/database.yml

```diff
@@ -19,14 +19,14 @@ default: &default
   encoding: unicode
   username: <%= ENV['OSBB_DATABASE_USERNAME'] %>
   password: <%= ENV['OSBB_DATABASE_PASSWORD'] %>
-  host: localhost
```

Comment:
pls stop editing this file at your own will
configuration is set for every one
communicate with a team of how to use common setup not your personal